src/Components/SuperAdmins/Form/index.jsx

```diff
@@ -1,13 +1,86 @@
-import React from 'react';
-import { useState, useEffect } from 'react';
+import React, { useState, useEffect } from 'react';
+import { useParams, useHistory } from 'react-router-dom';
 import styles from './super-admins.module.css';
 
-const Form = (props) => {
-  if (!props.showForm) {
-    return null;
-  }
+const Form = () => {
+  const [superAdmins, setSuperAdmins] = useState([]);
+  const [isEdit, setIsEdit] = useState(false);
+  const { id } = useParams();
+  const history = useHistory();
 
-  const isPut = props.method === 'PUT';
+  const getSuperAdmins = async () => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/api/super-admins`);
+      const data = await response.json();
+      setSuperAdmins(data.data);
+    } catch (error) {
+      console.error(error);
+      throw new Error('An error has occurred, cannot get the Super Admins');
+    }
+  };
+
+  useEffect(() => {
```

Comment:
Could you use a single useEffect instead of having 3?

Reply:
I will try!